lib/movie_list_widgets/movie_list.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+import '/screens/movie_home_page.dart';
+import '/constants/constants.dart';
+
+import '/data_classes/movie.dart';
+
+class MovieList extends StatelessWidget {
+  final List<Movie> moviesList;
+  static const double movieBoxSize = 125;
+  static const double titleSize = 20;
+  static const int overviewMaxLines = 3;
+  static const double elevationValue = 5;
+
+  const MovieList({
+    super.key,
+    required this.moviesList,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView.builder(
+      itemCount: moviesList.length,
+      itemBuilder: (
+        context,
+        index,
```

Comment:
As a recommendation add the type of the parameters (BuildContext, int)

Reply:
fixed